Automattic/jetpack
# Upgrade @wordpress/components package to 9.2.3

This was raised in https://github.com/Automattic/jetpack/pull/14927.

#### Changes proposed in this Pull Request:

* Upgrades our version of the @wordpress/components package to 9.2.3
* Updates the components we use from this package to be aligned with the new package.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Update to existing features

#### Testing instructions:
* Try adding the following blocks and check you can still edit their media:
* Simple Payments
* Tiled Gallery
* Slideshow

#### Proposed changelog entry for your changes:

* Upgrade @wordpress/components to 9.2.3
